src/macaron/slsa_analyzer/checks/detect_malicious_metadata_check.py

```diff
@@ -141,16 +141,15 @@ class MaliciousMetadataFacts(CheckFacts):
 
 
 class DetectMaliciousMetadataCheck(BaseCheck):
-    """This check analyzes the metadata of the pypi package based on seven heuristics."""
+    """This check analyzes the metadata of a package based on reports malicious behavior."""
 
     def __init__(self) -> None:
         """Initialize a check instance."""
         check_id = "mcn_detect_malicious_metadata_1"
-        description = "Check whether the features of package adhere to the heuristics."
-        super().__init__(
-            check_id=check_id,
-            description=description,
-        )
+        description = """This check analyzes the metadata of a package based on reports malicious behavior.
+        Supported ecosystem: PyPI.
+        """
+        super().__init__(check_id=check_id, description=description, eval_reqs=[])
```

Comment:
nit: we can add a trailing comma to the parameters list to simplify this diff.

Reply:
I think this is already very easy to read because the number of parameters is very small.

Reply:
I think what I meant was that we are introducing extra diff when it's not necessary to the commit content (which may affect the ability to git blame and find the original commit/PR that introduce the semantic of this block). I think this is perfectly fine in terms of aesthetic. 
However, I don't have a strong opinion on this so it could be left as it is.

Reply:
Black already has rules for handling such cases and let's avoid opinionated discussions. That's one one of the reasons we have so many linters in this project to let tools enforce conventions :slightly_smiling_face: